tests/unittests/BackendTestUtils.cpp

```diff
@@ -932,7 +932,7 @@ FunctionTensorPair createAndInitBasicFCNet(PlaceholderBindings &bindings,
   auto *fc = F->createFullyConnected(bindings, "fc", tr, 16);
   auto *rl0 = F->createRELU("relu", fc);
   auto *fc2 = F->createFullyConnected(bindings, "fc2", rl0, 8);
-  auto *rl1 = F->createRELU("relu", fc2);
+  auto *rl1 = F->createRELU("relu2", fc2);
```

Comment:
What's the context for this change?

Reply:
This was needed since this broke node name uniqueness with line 933.
It was already fixed in Graph::addNode*() a while back so I've reverted this change (though you may want it anyways for good order)